### DISCUSSION TITLE
Can mydumper be used at the same time with --stream and --rows

### DISCUSSION THREAD

**Discussion by HelioCampos** [mydumper/mydumper#1102] [General]
Hi team,

Such a wonderful tool! Congrats on it. I want to perform a migration of a database as fast as possible. For this I was thinking on using the parallel options of mydump like `--rows` but I also wanted to avoid saving files on the local disk with `--stream`. Is it possible to use both at the same time together with `myloader` to stream the process? How would `myloader` understand what I'm piping in to it?

Example:

mydumper --rows --stream | myloader

**Comment by davidducos** [comment]
Hi @HelioCampos, 
you can use MyDumper as in your example, but you missed the --stream on myloader:
```
mydumper --rows --stream | myloader --stream
```
Take into account that mydumper and myloader will write on disk the files always, even with --stream. The difference is that, after the file has been streamed, it will be deleted.
More info in: https://www.percona.com/blog/mydumper-stream-implementation/
